### ISSUE TITLE
Why did you make it so hard/confusing to customize the look and feel of this library

### ISSUE THREAD

**Issue by mihaimyh** [microsoft/fluentui-blazor#1176]
This library has a real potential, but I find it really hard/confusing when it comes to changing the default colours/fonts, the styling in general.

I would love a global way to customize the main styles of the entire application (primary, secondary colours, fonts, spacing, etc). Is it possible, or am I missing something?

**Comment by vnbaaij** [comment]
Yes, seems you are missing something. We are implementing the Fluent Design System. That system is a lot stricter on what fonts, size and color you 'should' use. It is quite different from something like Bootstrap. For example, Fluent doesn't have a real notion of a secondary color. 

We follow those strict guidelines in the default settings that we implement. 
If you want to deviate from what is prescribed, that is entirely possible but only by using the Design Tokens. We are still thinking about and working on on ways to make that a bit easier. For example with the `FluentDesignTheme` component we released last week.
Please take a look at the mentioned Fluent Design guidelines. 

Ps. We are still on Fluent 1. Fluent 2 will come with the next version of the web components

**Comment by kzu** [comment]
Takes some getting used to @mihaimyh, but unless you have a dedicated design team, it's a such a breeze not having to think about colors and fonts :). Not being a designer myself, obviously. Coming from bootstrap, it took some adapting, but now I only define my accent color and call it a day (design-wise).
